docs/snippets/update/db/update_db_2.5-3.3.md

```diff
@@ -23,3 +33,112 @@ Check the Location ID of the "Components" Content item and set it as a value of
 
 If you are upgrading between [[= product_name_com =]] versions,
 add the `content/read` Policy with the Owner Limitation set to `self` to the "Ecommerce registered users" Role.
+
+### Ibexa Open Source
+
+If you're upgrading to Ibexa Open Source v3.3 and have no access to `ibexa/installer` package, run the following SQL commands:
+
+=== "MySQL"
+
+    ``` sql
+    START TRANSACTION;
+    DELETE FROM `ezsite_data` WHERE `name` IN ('ezpublish-version', 'ezpublish-release', 'ezplatform-release');
+    INSERT INTO ezsite_data (`name`, `value`) VALUES ('ibexa-release', '3.3.17');
```

Comment:
> We need to keep this in sync with the latest 3.3.x release, right @alongosz?

True, for doc it should probably say `3.3.16`. Obviously it creates a bit of maintenance "hiccup". Maybe it's an overkill and it should be just 3.3 until it's really used?

Reply:
Changed as described, too much of a hassle to remember about this, especially that we don't use it yet.

Reply:
To be honest we should make it part of the upgrade script, if it's really necessary to keep this in database.

:+1: for the current version.

Reply:
> To be honest we should make it part of the upgrade script, if it's really necessary to keep this in database.
> 
> +1 for the current version.

It's rather maintenance around what was already there since eZ Publish 3.x. As said - we don't use it yet.